### DISCUSSION TITLE
Bug in rendering?

### DISCUSSION THREAD

**Discussion by ywteh** [RomRider/apexcharts-card#813] [Q&A]
Hi, I've been using apexcharts to plot some information, it's great! but I seem to be encountering some bug?  Attached is two figures, one when I'm trying to edit the card (looks great), and one when I'm not editing the card (looks weird). I'm trying to stack the lines.
<img width="434" alt="Screenshot 2024-11-13 at 12 38 50" src="https://github.com/user-attachments/assets/72ea8369-6141-4012-b44e-24a9639048fb">
<img width="298" alt="Screenshot 2024-11-13 at 12 39 05" src="https://github.com/user-attachments/assets/383bb614-714d-4648-a235-408109e3e65f">
 
Can anyone help me find what the problem is? cheers, -yw

 The card specs is as follows:
 `type: custom:apexcharts-card
graph_span: 4h
header:
  show: true
  title: Heat demand
  show_states: true
  colorize_states: true
apex_config:
  legend:
    show: false
stacked: true
all_series_config:
  stroke_width: 2
  group_by:
    func: avg
    duration: 5min
  fill_raw: last
series:
  - entity: sensor.kitchen_heat_demand
    name: kitchen
    color: dark-grey
  - entity: sensor.ground_zero_heat_demand
    name: ground_zero
    color: grey
  - entity: sensor.office_heat_demand
    name: office
    color: "#6100F2"
  - entity: sensor.tv_room_heat_demand
    name: tv_room
    color: "#F2C509"
  - entity: sensor.music_room_heat_demand
    name: music_room
    color: "#00A99D"
  - entity: sensor.studio_heat_demand
    name: studio
    color: orange
  - entity: sensor.family_bath_heat_demand
    name: family_bath
    color: turquoise
  - entity: sensor.landing_heat_demand
    name: landing
    color: grey
  - entity: sensor.library_heat_demand
    name: library
    color: "#00A99D"
  - entity: sensor.qq_room_heat_demand
    name: qq_room
    color: green
  - entity: sensor.pp_room_heat_demand
    name: pp_room
    color: blue
  - entity: sensor.momo_room_heat_demand
    name: momo_room
    color: "#F7965A"
  - entity: sensor.tip_top_heat_demand
    name: tip_top
    color: "#B6321C"
  - entity: sensor.top_bath_heat_demand
    name: top_bath
    color: "#B6321C"
  - entity: sensor.top_garden_heat_demand
    name: top_garden
    color: "#B6321C"
`

**Comment by TheSerapher** [comment]
I think what you are experiencing is is similar to what I had: https://github.com/RomRider/apexcharts-card/issues/144#issuecomment-2421899073

No solution for it though, I just live with it for now.
